webprojects/amo.md

```diff
@@ -0,0 +1,93 @@
+# AMO: how to localize and test
+
+## Product at a glance
+
+AMO refers to https://addons.mozilla.org/. It is one of Mozilla’s biggest web properties, with 12 million users per month. The product consists three key components that require localization:
+
+* [AMO server](https://pontoon.mozilla.org/projects/amo) is the backend component of AMO
+* [AMO frontend](https://pontoon.mozilla.org/projects/amo-frontend/) covers the top page of `about:addons` and the user-facing pages of addons.mozilla.org on Firefox for Android.
+* [Disvoery Pane](https://pontoon.mozilla.org/projects/amo-frontend/) is home of curated addons available on `about:addons`.
+
+Product updates are deployed every Thursday to production. Localizable strings can be extracted on Tuesdays, and changes landed by 5pm UTC each Tuesday go live on Thursday. To report issues, go to: https://github.com/mozilla/addons-server/issues/new.
+
+Now let’s take a look of these components in greater details.
+
+## AMO and AMO frontend
+
+### [addons-server](https://pontoon.mozilla.org/projects/amo)
+
+**Addons-server** is the default front-end for desktop users of AMO. It is also the API backend and home to the developer hub and reviewer tools.
+
+### [addons-frontend](https://pontoon.mozilla.org/projects/amo)
+
+**Addons-frontend** is the new frontend and default frontend for mobile user-agents. It will eventually replace the **addons-server** frontend for user-pages. **Addons-frontend** currently provides an AMO app (amo.po).
+
+### Test the server and the frontend
+
+The strings will be available on Dev or Staging server depending on when they were committed. See [this section](https://github.com/mozilla-l10n/localizer-documentation/blob/2f065c6a15208cf30ec0a8d440a399a5b5424d39/webprojects/amo.md#frequencies-of-newly-extracted-strings-and-landing-of-newly-localized-strings) for details.
+
+* Dev https://addons-dev.allizom.org/
+* Staging https://addons.allizom.org/
+* Production https://addons.mozilla.org/
+
+The hosts for the new frontend are the same as **addons-server**, except to be routed to the new frontend, you either need:
+
+* Setting a “mamo“ cookie with the value `on` by clicking _View mobile site_ in the footer.
+* A Mobile user-agent string.
+
+## Disco Pane app and editorial content
+
+The AMO Frontend (addons-frontend) repo is also home to the Disco Pane, short for Discovery Pane app (disco.po). This is the site that contains curated addons available by typing `about:addons` on the Awesome Bar. Another way to get to the same place is by clicking _Tools_ on the navigation, select _Add-ons_ in the dropdown menu, and click _Get Add-ons_ on the left panel). While this looks like it’s part of Firefox, it’s actually a webpage served from [here](https://discovery.addons.mozilla.org).
```

Comment:
"in the address bar"
Again, asterisks for Italic, not underscores.
Are you sure about the menu instructions?